src/api/app/controllers/webui/project_controller.rb

```diff
@@ -13,7 +13,7 @@ class Webui::ProjectController < Webui::WebuiController
 
   before_filter :lockout_spiders, :only => [:requests, :rebuild_time, :buildresults, :maintenance_incidents]
 
-  before_filter :require_login, :only => [:create, :toggle_watch, :destroy, :new]
+  before_filter :require_login, :only => [:create, :toggle_watch, :destroy, :new, :new_incident_request, :new_release_request, :new_package_branch, :new_package]
```

Comment:
Why not the except? Then someone has to add it explicitly if a login is not required!